examples/gym_ex/v2/bandit_proxy_env.py

```diff
@@ -90,15 +131,17 @@ def receive_percept_message(self) -> Message:
             raise ValueError("Missing envelope.")
 
     def message_to_percept(self, message: Message) -> Feedback:
+        """
+        Transform the message received from the real environment into observation, reward, done, info.
+
+        This is an implementation of the abstract method in ProxyEnv.
+
+        :param: the message received as a response to the action performed in apply_action.
+        :return: the standard feedback (observation, reward, done, info) of a gym environment.
+        """
         observation = message.get("observation")
         done = message.get("done")
         info = message.get("info")
         reward = message.get("reward")
 
         return observation, done, reward, info
-
-    def render(self, mode='human'):
```

Comment:
I think they should stay? This is an implementation of the `gym.Env` abstract method